descartes_moveit/package.xml

```diff
@@ -14,26 +14,18 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>rosconsole_bridge</build_depend>
-  <build_depend>moveit_core</build_depend>
-  <build_depend>tf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>descartes_core</build_depend>
-  <build_depend>descartes_trajectory</build_depend>
-  <build_depend>moveit_ros_planning</build_depend>
-  <build_depend>pluginlib</build_depend>
-  <build_depend>eigen</build_depend>
```

Comment:
eigen wasn't previously in the run_depend, therefore it should be <build_depend> because it doesn't have a dependency at run-time.